package.json

```diff
@@ -49,20 +49,22 @@
   },
   "browser": "lib/browser.js",
   "dependencies": {
-    "asn1.js": "=4.9.1",
-    "asn1.js-rfc3280": "=4.0.0",
-    "asn1.js-rfc5280": "=2.0.0",
-    "bitcore-lib": "^0.15.0",
+    "asn1.js": "^5.0.1",
+    "asn1.js-rfc3280": "^4.0.0",
+    "asn1.js-rfc5280": "^3.0.0",
     "bs58": "^4.0.1",
-    "jsrsasign": "^8.0.4",
-    "protobufjs": "=5.0.1"
+    "jsrsasign": "^8.0.12",
+    "protobufjs": "=5.0.3"
+  },
+  "peerDependencies": {
+    "bitcore-lib": "=0.16.0"
   },
   "devDependencies": {
```

Comment:
Do the latest versions of all of these dependencies support the old devices we need to run on?

Reply:
The latest versions do not appear to be necessary for proper functionality. They were updated by the BitPay devs as what appears to be housekeeping and not to enable any new features or fix any bugs.